crates/relayer/src/chain/namada/tx.rs

```diff
@@ -284,6 +315,81 @@ impl NamadaChain {
         }
     }
 
+    fn estimate_fee(
+        &self,
+        mut tx: tx::Tx,
+        args: &TxArgs,
+        signing_data: &signing::SigningTxData,
+    ) -> Result<(Address, u64, f64), Error> {
```

Comment:
```suggestion
    ) -> Result<Option<(Address, u64, f64)>, Error> {
```
Return an option, since nodes should be able to disable the dry run tx rpc method. Also, let's move away from a tuple here, especially considering this is a public facing API. Let's create a struct that houses all of these fields and gives them a meaningful name.